source/_integrations/onvif.markdown

```diff
@@ -37,6 +37,8 @@ You can configure specific FFmpeg options through the integration options flow b
 | RTSP transport mechanism | RTSP transport protocols. The possible options are: `tcp`, `udp`, `udp_multicast`, `http`. |
 | Extra FFmpeg arguments | Extra options to pass to `ffmpeg`, e.g., image quality or video filter options. More details in [`ffmpeg` component](/integrations/ffmpeg). |
 
+You can also configure which HTTP authentication type (`basic` or `digest`) is used when retrieving snapshots. The default is `digest`.
```

Comment:
Would you mind making this part of the same table as the other options above? Similar to RTSP transport mechanism.

Reply:
The reason I kept it out of the table is that the other two options are ffmpeg options and the text before them calls them such, so I felt it made sense to separate them.